python/feathub/processors/flink/flink_processor.py

```diff
@@ -233,6 +240,16 @@ def materialize_features(
                 # a local registry.
                 join_tables.update(self._get_join_tables(resolved_feature_descriptor))
 
+            metric_descriptor = None
+            if self.metric_store is not None:
+                metric_descriptor = self.metric_store.materialize_metrics_if_any(
```

Comment:
Most existing methods named `*materialize*` actually starts a job to ETL metrics. It is inconsistent and confusing to have a method named `materialize_metrics_if_any` that only creates the `MaterializationDescriptor`.

Would it be simpler to rename it as `create_metric_materialization_descriptor`?